### PR TITLE
WAVE2-912-adding FE to degree details page

### PR DESCRIPTION
**Pull request by nindusaini**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc [hunk 1]
```diff
@@ -979,43 +979,6 @@ function ucgy_fgs_field_default_field_bases() {
...
-        'target_bundles' => array(
-          'faculty' => 'faculty',
-        ),
-      ),
-      'target_type' => 'faculty',
-    ),
-    'translatable' => 0,
-    'type' => 'entityreference',
-  );
-
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc]: Finally these are removed!!! They kept coming back from the dead for a while.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/views-view--fgs-grad-degree-supervisors.tpl.php [hunk 2]
```diff
@@ -27,64 +27,102 @@
...
+      <?php endif; ?>
     </div>
-  <?php endif; ?>
+  </div>
+  <?php
+}
+?>
+
+<div class="row divider"><div class="col-sm-12"><hr></div></div>
+
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/views-view--fgs-grad-degree-supervisors.tpl.php]: Not sure where in this view filter code it's going wrong, but the filter isn't working.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module [hunk 3]
```diff
@@ -80,3 +78,50 @@ function ucgy_fgs_form_alter(&$form, &$form_state, $form_id) {
...
 }
+
+/**
+ * Implements hook_page_build
+ */
+function ucgy_fgs_page_build(&$page){
+  //we need to add CSS and JS for ucgy_fgs_all_programs view only for degree details page, hence following check
+  $node = menu_get_object();
+  if($node->type == 'fgs_graduate_degree'){
+    drupal_add_js(drupal_get_path('module', 'ucgy_fgs_all_programs') . '/js/ucgy_fgs_all_programs.js');
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.module]: The js doesn't seem to be working. I can't select a related program (clicking on the star does nothing).

**Comment by dpfitzsi** [comment]
Merged in to wave2-910-912-914-974-fe-compare branch.

**Pull request closed without merging**